Core/Config.h

```diff
@@ -31,6 +31,12 @@ extern const char *PPSSPP_GIT_VERSION;
 
 extern bool jitForcedOff;
 
+// should this be here?!
+// for Config.iShowFPSCounter
+#define SHOW_FPS_COUNTER 1 << 1
+#define SHOW_SPEED_COUNTER 1 << 2
+#define SHOW_BATTERY_PERCENT 1 << 3
```

Comment:
When using #define with expressions, you need to use parentheses. Like:

`#define SHOW_FPS_COUNTER (1 << 1)`

Consider

`int x = flags & SHOW_FPS_COUNTER;`

the result is otherwise

`int x = flags & 1 << 1;`  which actually evaluates as `int x = (flags & 1) << 1;` due to the low precedency of shift operators.

Reply:
Ah, I forgot about that for bit masks, will do

Reply:
Addressed in https://github.com/hrydgard/ppsspp/pull/16851/commits/ca154879f05e829c6c333c8f27c1ea0f1f65d4d2